dockerfiles/docker-compose.for-server-image.yaml

```diff
@@ -4,7 +4,7 @@ version: '3.5'
 
 services:
   temporal-server:
-    image: temporaliotest/auto-setup:latest
+    image: temporaliotest/auto-setup:${IMAGE_TAG:-latest}
```

Comment:
Where does the dash come from?

Reply:
https://docs.docker.com/compose/environment-variables/env-file/#interpolation

Reply:
Ah, got it. That syntax is, well, it's a syntax.